package.json

```diff
@@ -71,6 +71,7 @@
     "build": "tsc --project tsconfig.json",
     "init-test": "rm nonce.db; rm sudo.lock;",
     "test": "jest",
+    "test-run": "export JEST_HTML_REPORTERS_PUBLIC_PATH=reports/html-report-seq; rm nonce.db ; rm sudo.lock;  node --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules node_modules/jest/bin/jest.js --forceExit ",
```

Comment:
use to run specific tests localy
yarn test-run some.test.ts - t 'test foo should be true'